src/display/api.js

```diff
@@ -1124,6 +1133,18 @@ class PDFPageProxy {
     return this.annotationsPromise;
   }
 
+  /**
+   * @param {GetAnnotationsParameters} params - Annotation parameters.
+   * @returns {Promise<Array<any>>} A promise that is resolved with an
+   *   {Array} of the annotation objects.
+   */
+  getJSActions() {
+    if (!this.jsActionsPromise) {
```

Comment:
This should be `this._jsActionsPromise` instead, and please also remember to reset it with `this._jsActionsPromise = null;`-lines after all existing `this.annotationsPromise = null;`-lines in this file.